nekoyume/Assets/_Scripts/BlockChain/ActionHandler.cs

```diff
@@ -118,9 +118,9 @@ private static void UpdateGoldBalanceState(GoldBalanceState goldBalanceState)
             States.Instance.SetGoldBalanceState(goldBalanceState);
         }
 
-        private void UpdateAvatarState(AvatarState avatarState, int index)
+        private async void UpdateAvatarState(AvatarState avatarState, int index)
```

Comment:
We can just return `Task` instead of `async` keyword if there is no statement other than `await`.

```csharp
        private Task UpdateAvatarState(AvatarState avatarState, int index)
            => States.Instance.AddOrReplaceAvatarState(avatarState, index);
```